node/actors/bft/src/leader/replica_commit.rs

```diff
@@ -58,6 +64,13 @@ impl StateMachine {
         let message = signed_message.msg;
         let author = &signed_message.key;
 
+        consensus
+            .validator_set
+            .index(author)
+            .ok_or(Error::NonValidatorSigner {
+                signer: author.clone(),
+            })?;
```

Comment:
Nit. Add a comment saying what you are checking.

Reply:
Fixed.